### ISSUE TITLE
regression: ICE is not supported for this key

### ISSUE THREAD

**Issue by BoxyUwU** [rust-lang/rust#130573]
- https://crater-reports.s3.amazonaws.com/beta-1.82-1/beta-2024-09-05/reg/empty_type_derive-0.2.3/log.txt
  - https://crater-reports.s3.amazonaws.com/beta-1.82-1/beta-2024-09-05/gh/dpl0a.type-aliasing-bug/log.txt
  - https://crater-reports.s3.amazonaws.com/beta-1.82-1/beta-2024-09-05/gh/Luminoth.mazes-for-programmers/log.txt
```
[INFO] [stdout] error: internal compiler error: compiler/rustc_middle/src/query/plumbing.rs:653:5: `tcx.extern_crate(LocalCrate)` is not supported for this key;
[INFO] [stdout]                                 hint: Queries can be either made to the local crate, or the external crate. This error means you tried to use it for one that's not supported.
[INFO] [stdout]                                 If that's not the case, extern_crate was likely never assigned to a provider function.
[INFO] [stdout] 
[INFO] [stdout] 
[INFO] [stderr] thread 'rustc' panicked at compiler/rustc_middle/src/query/plumbing.rs:653:5:
[INFO] [stderr] Box<dyn Any>
```

**Comment by fmease** [comment]
Should be fixed by #130275. Nominating for beta backport.

**Comment by compiler-errors** [comment]
lmao fmease beat me to it

**Comment by apiraino** [comment]
Closing a FIXED since #130275 has been merged
